core/src/main/java/io/aiven/klaw/helpers/db/rdbms/SelectDataJdbc.java

```diff
@@ -1437,66 +1438,43 @@ public Optional<ProductDetails> selectProductDetails(String name) {
   }
 
   public boolean existsKafkaComponentsForEnv(String env, int tenantId) {
-    List<Supplier<Boolean>> list =
-        List.of(
+    return Stream.<Supplier<Boolean>>of(
```

Comment:
I like the code but not very readable/easily understandable sometimes. Does it have any impact on performance ?

Reply:
this PR is not about performance it is about making code a bit shorter.
The perf problem was solved in prev PR which made it lazy

if you have some doubts about concrete places, can you share a bit more, perhaps we can do something about it

Reply:
Personally I am not a big fan of making code shorter always. Prefer to make it readable/ easily understandable for most of the developers.

Reply:
The main change here is to replace the cycle 
```java
for (var elem : list) {
      if (elem.get()) {
        return true;
      }
    }
    return false;
```
with usage of streams. Do you think cycle is more readable or what is your point about this change?

Reply:
@snuyanzin got it. It's ok.